bundles/core/org.eclipse.smarthome.core.test/src/test/groovy/org/eclipse/smarthome/core/internal/i18n/LocationProviderOSGITest.groovy

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.internal.i18n
+
+import static org.hamcrest.CoreMatchers.*
+import static org.junit.Assert.*
+import static org.junit.matchers.JUnitMatchers.*
+
+import org.eclipse.smarthome.core.i18n.LocationProvider
+import org.eclipse.smarthome.test.OSGiTest
+import org.junit.Before
+import org.junit.Test
+import org.osgi.service.cm.ConfigurationAdmin
+
+/**
+ * The {@link LocationProviderOSGiTest} tests the basic functionality of the {@link LocationProvider} OSGi service.
+ *
+ * @author Stefan Triller - Initial contribution
+ */
+class LocationProviderOSGiTest extends OSGiTest {
```

Comment:
Sorry, didn't notice that - but you are certainly right - @triller-telekom could you migrate that to a Java test please?